README.md

```diff
@@ -95,13 +95,14 @@ Moreover, as of 3.8.0, a validator can be an Object or Class with a `validate` f
 
 ```js
 import  { inject as service } from '@ember/service';
+import fetch from 'fetch';
 
 export default class PersonalNoValidator {
-  @service ajax;
 
   async validate(key, newValue, oldValue, changes, content) {
     try {
-      await this.ajax.post('/api/personal-no/validation', { data: newValue });
+      await fetch('/api/personal-no/validation', body: JSON.stringify({ data: newValue }));
```

Comment:
We can probably add `method: POST` to fetch?

Reply:
Indeed, I also have added the appropriate header.